### ISSUE TITLE
Could we fail on unknown Attribute?

### ISSUE THREAD

**Issue by pashashocky** [tefra/xsdata#597]
Could we please add another config option in addition to `fail_on_unknown_properties` to fail on unknown `Attribute`? 

I don't currently see that functionality and I've experimented by adding random attributes to my parser and they didn't crash or raise a `TypeError`. 

I am using `attrs` plugin with the `kw_only` flag set to `true` in the config. 

I would be expecting similar behaviour to this:

```
In [1]: import attr

In [2]: @attr.s(frozen=True, slots=True, kw_only=True)
   ...: class Test:
   ...:     a: int = attr.ib()
   ...:     b: str = attr.ib()

In [3]: Test(**{'a': 3, 'b': 'test', 'c': 'tent'})
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-10-715d11a185fb> in <module>
----> 1 Test(**{'a': 3, 'b': 'test', 'c': 'tent'})

TypeError: __init__() got an unexpected keyword argument 'c'
```

**Comment by tefra** [comment]
merged #598 thank you for your contribution @pashashocky

**Comment by pashashocky** [comment]
@tefra thank you, was a pleasure - and appreciate the tips!
